synapse/lib/types.py

```diff
@@ -1598,7 +1598,7 @@ def _normPyStr(self, valu):
 
         return norm, info
 
-taxonre = regex.compile('\w+')
+taxonre = regex.compile('\\w+')
```

Comment:
Uhh, isn't this changing the meaning of the regex to mean something very different?

Reply:
These are using deprecated escape sequences in string types which will eventually become a python syntax error.

Reply:
```
>>> '\w+' == '\\w+'
True
```